src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -73,13 +73,14 @@ const SignedChatScreen = () => {
         style={styles.chatContainer}
         data={updatedChats}
         inverted={true}
-        initialNumToRender={20}
+        initialNumToRender={10}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `initialNumToRender={20}` to `initialNumToRender={10}` reduces the number of chat items initially rendered. This might improve performance by reducing initial rendering load, but it could also negatively impact user experience if users expect to see more than 10 items at once when they open the chat. Please ensure this change aligns with your application's user experience expectations.
```diff
-        initialNumToRender={20}
+        initialNumToRender={10}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->